CHANGES.txt

```diff
@@ -5,7 +5,7 @@ CHANGELOG
 3.4.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Add traverse support (#483)
```

Comment:
Maybe you could add a link to https://docs.pylonsproject.org/projects/pyramid/en/1.9-branch/narr/hybrid.html#using-the-traverse-argument-in-a-route-definition and be more explicit (ex. mention hybrid traversal/url dispatch)